perf/range_conversion.cpp

```diff
@@ -0,0 +1,100 @@
+// Range v3 library
+//
+//  Copyright Eric Niebler 2019-present
+//
+//  Use, modification and distribution is subject to the
+//  Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+// Project home: https://github.com/ericniebler/range-v3
+//
```

Comment:
BSL is "free for all uses, and you don't even need to include this license but you do have to include copyright." Speaking of which, this code is also yours so I'll add a copyright notice for you unless you say otherwise.

Reply:
SGTM (I use this code in classrooms).